docs/RECIPES.md

```diff
@@ -137,3 +137,33 @@ EOF
 ```shell
 oc apply -f preflight.yaml
 ```
+
+## Container Policy
+These examples are shown using the Container policy against a container image
+(e.g. `preflight check container <image>`). Container policy only runs as a binary on your workstation. Check the latest
+[release](https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases) for the binary that matches your operating system.
+
+You will also need:
+- Your container image published to a container registry
+- A Certification Project ID of the project that was set up in Red Hat Partner Connect
+- A Partner Connect API Key
```

Comment:
Sudo. This is the place to mention it.

Reply:
@bcrochet Let me know your thoughts on the wording.

Reply:
Removed this for now, since the checks that need `sudo` are influx.